### ISSUE TITLE
Allow assigning a stick axis to different axis

### ISSUE THREAD

**Issue by randomname42** [AntiMicroX/antimicrox#662]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe.

Some games don't let you configure what is action is performed on the analog axis. In particular FPS games which don't let you chose what sticks/axis are set to strafe, turn, move forward/back, look up/down. For example if you want move and turn on the same stick, you cannot set that in the game. You're forced to have move and strafe together on the same stick.

I thought AntiMicroX would be able assign a stick axis to different stick axis (which would solve the problem) but it does not look like there is a way to do it. Unfortunately that leaves no way to set controls how I want them to be.

### Describe the solution you'd like

Allow assigning any joystick/analog axis to a specific axis on a specific stick.

For example, assigning _right_ stick up/down to _left_ stick up/down--making it so that when you move the _left_ stick up/down, the game will think you are moving the _right_ stick up/down.

Basically, I'm thinking that in the key assignment window there could be an extra tab besides Keyboard and Mouse, for assigning a stick axis.

**Comment by pktiuk** [comment]
Issue similar to https://github.com/AntiMicroX/antimicrox/issues/65

## Mapping to gamepad buttons  or other axes

For now, I can say that mapping to another gamepad buttons/axes is out of scope of antimicrox.

> AntiMicroX is a graphical program used to map gamepad keys to keyboard, mouse, scripts and macros.

Not for another gamepads or virtual devices (at least not for now).   
There are already some apps which let you map your gamepad to another one like x360ce
